.github/workflows/integration.yml

```diff
@@ -63,9 +63,10 @@ jobs:
         mkdir -p data/elastic/
         chmod -R 777 data/
         cp .env.integration .env
-        echo AZURE_OPENAI_ENDPOINT=${{ secrets.AZURE_OPENAI_ENDPOINT }} >> .env
-        echo AZURE_OPENAI_API_KEY=${{ secrets.AZURE_OPENAI_API_KEY }} >> .env
-        echo AZURE_OPENAI_MODEL=azure/gpt-4-turbo-2024-04-09 >> .env
+        echo AZURE_OPENAI_ENDPOINT_4T=${{ secrets.AZURE_OPENAI_ENDPOINT }} >> .env
+        echo AZURE_OPENAI_API_KEY_4T=${{ secrets.AZURE_OPENAI_API_KEY }} >> .env
+        echo EMBEDDING_AZURE_OPENAI_ENDPOINT=${{ secrets.AZURE_OPENAI_ENDPOINT }} >> .env
+        echo EMBEDDING_OPENAI_API_KEY=${{ secrets.AZURE_OPENAI_API_KEY }} >> .env
```

Comment:
we need to set the same variables twice: once for the embedding model and once for the relevant chat model